test/controllers/users_controller_test.rb

```diff
@@ -0,0 +1,74 @@
+# frozen_string_literal: true
+
+require 'test_helper'
+
+class UsersControllerTest < ActionDispatch::IntegrationTest
+  setup do
+    sign_in users(:serhat)
+    @user = users(:john)
+  end
+
+  test 'should get index' do
+    get users_path
+    assert_response :success
+    assert_select '#add-button', t('users.index.new_user_link')
+  end
+
+  test 'should get show' do
+    get user_path(@user)
+    assert_response :success
+  end
+
+  test 'should get new' do
+    get new_user_path
+    assert_response :success
+  end
+
+  test 'should create user' do
```

Comment:
Bu test Circle'da fail etmiş. İlk başta Circle'ın IP adresi MERNİS'te tanımlı olmadığı için diye düşündüm ama yerelimde deneyince de başka bir hata aldım. Çünkü sanırım `33998445768` gerçek bir TC numarası değil. Wiki'den demo TC'leri alabilirsin. Eğer bunlarda yeterli olmazsa söylersin yenilerini eklerim.

Bu test senin yerelinde geçiyor muydu? Yoksa aldığın 403'ü ignore mu ettin? VPN'siz test edersen alacağın gerçek hatayı yakalayamazsın. @ecylmz tüm geliştiricilere VPN açmıştı. Eğer MERNİS veya YÖKSİS ile ilgili bir test çalıştırıyorsan bunu VPN'e bağlanarak yapmalısın ki gerçek hataları yakalayabil. Hatta genel bir pratik olarak nokul'da geliştirme yaparken BAUM VPN'e hep bağlı ol ki üniversitedeymiş gibi çalışabilesin.

Ne yazık ki şimdilik CI'da YÖKSİS veya KPS'i test etmemizin şuan için bir yolu olmadığı için bu testi skip etmen gerekiyor. Yerelinde bunun geçtiğinden VPN'e bağlanıp emin olduktan sonra CI için skip edebilirsin. Örneğin: https://github.com/omu/nokul/blob/develop/test/jobs/kps_address_job_test.rb#L24

Reply:
@dilara Buna dönüş yapmadın?